src/Storefront/Resources/snippet/de_DE/storefront.de-DE.json

```diff
@@ -278,8 +278,14 @@
     "addressesSetAsDefaultBillingAction": "Als Standard-Rechnungsadresse verwenden",
     "addressesTitleCurrentShippingAddress": "Aktuelle Lieferadresse",
     "addressesTitleCurrentBillingAddress": "Aktuelle Rechnungsadresse",
+    "addressesTitleShippingAddress": "Lieferadresse",
+    "addressesTitleBillingAddress": "Rechnungsadresse",
+    "addressesChangeAddress": "Adresse ändern",
+    "addressModalCancel" : "Abbrechen",
     "addressesContentItemActionEdit": "Adresse bearbeiten",
     "addressesContentItemActionDelete": "Adresse löschen",
+    "addressNotFound": "Keine Adresse gefunden",
```

Comment:
This snippet is not used. Was this supposed to be an empty-state for the search? I would appreciate that instead of just showing nothing